### ISSUE TITLE
iOS - Setting Stroke doesn't work on Shape

### ISSUE THREAD

**Issue by shwetsolanki** [react-native-community/art#30]
# Bug

I am unable to set stroke to a shape on iOS. Android it works fine. 

If I go to Shape.js and replace `stroke={extractColor(props.stroke)}` with `stroke={props.stroke}`, then I am able to see the stroke on iOS but the app crashes for Android

## Environment info

React native info output:

```
React Native Environment Info:
    System:
      OS: macOS 10.14.5
      CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
      Memory: 514.37 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.16.0 - /usr/local/bin/node
      Yarn: 1.16.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
      Android SDK:
        API Levels: 19, 23, 25, 26, 27, 28
        Build Tools: 23.0.1, 26.0.3, 27.0.3, 28.0.2, 28.0.3
        System Images: android-19 | Google APIs Intel x86 Atom, android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5692245
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.9 => 0.59.9 

```

Library version: 1.0.1

## Steps To Reproduce

1. Go to `./example/components/Heart.js`
2. Replace `<Shape d={HEART_SHAPE} fill={gradient} />` with `<Shape d={HEART_SHAPE} stroke={'#00ff00'} />`
3. `npx react-native run-ios`
...

Describe what you expected to happen:

1. Heart should be rendered with a green stroke

**Comment by smacgregor** [comment]
It appears to be a regression from replacing createReactNativeComponentClass with requireNativeComponent in

https://github.com/react-native-community/art/blob/master/lib/nativeComponents.js

Reverting back to createReactNativeComponentClass like the package inside React Native core restores setting the stroke.

(I also needed https://github.com/react-native-community/art/pull/28)

**Comment by shwetsolanki** [comment]
@smacgregor Could you check my PR #31, I have updated the implementation of Android such that it accepts hex color which can be converted to Color object

**Comment by shwetsolanki** [comment]
@Esemesek @smacgregor could you look into the PR?

**Comment by ollyfg** [comment]
Just FYI, I also experienced this issue. Forking @shwetsolanki 's PR branch fixed the problem for me.  👌

**Comment by Esemesek** [comment]
Fixed in #31
